Parcial de preogramación 1/israel boksh parcial #1.cpp

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+ main ()
+{
+	
+	if ("num d.50 m" .50) ( num e"100*d");
+	else ("% d" d.50* 4);
+	
+	Printf("\n multiplicar la parte entere\n;");
+	break;
+	
+	if(4 d*a=5 , d* a=11);
+	else("% d",num*4);
+	
+	printf("\n no es divisible\n,");
+	break;
+	
+	it(" numd.50 m" .50*4);

```

Comment:
¿it?
Es en serio

Reply:
![images](https://user-images.githubusercontent.com/31838625/30382602-2da78b2c-9865-11e7-9e16-b2d0cc72e235.jpg)